Compatibility with opensearch 2.19.0 (#191)
wazuh/wazuh-security-dashboards-plugin: README.md

<div style="text-align:center">
    <img src="https://wazuh.com/uploads/2022/05/Logo-blogpost.png"/>
</div>

> # Wazuh Security Dashboards Plugin

Reviewer: ```suggestion
# Wazuh dashboard Security Plugin
```

> Wazuh Security Dashboards Plugin is a fork of the OpenSearch Dashboards Security Plugin which incorporate changes to make it easier to use for Wazuh users. Our aim is to contribute back any work not tied specifically to Wazuh.

Reviewer: ```suggestion
Wazuh dashboard Security Plugin is a fork of the OpenSearch Dashboards Security Plugin which incorporates changes to make it easier to use for Wazuh users. Our aim is to contribute back any work not tied specifically to Wazuh.
```

> This plugin for Wazuh Dashboard adds a configuration management UI for the Wazuh Security features, as well as authentication, session management and multi-tenancy support to your secured cluster.

Reviewer: ```suggestion
This plugin for Wazuh dashboard adds a configuration management UI for the Wazuh Indexer Security features, as well as authentication, session management and multi-tenancy support to your secured cluster.
```

- [Features](#features)
- [Installation](#installation)
- [Contributing](#contributing)
- [Getting Help](#getting-help)
- [Security](#security)
- [License](#license)
- [Copyright](#copyright)

## Features

> * Wazuh Dashboard authentication for OpenSearch
> * Wazuh Dashboard session management
> * Wazuh Security configuration UI
> * Multi-tenancy support for Wazuh Dashboard
> * Wazuh audit logging configuration UI

Reviewer: ```suggestion
* Wazuh dashboard authentication for Wazuh indexer
* Wazuh dashboard session management
* Wazuh indexer Security configuration UI
* Multi-tenancy support for Wazuh dashboard
* Wazuh indexer audit logging configuration UI
```

## Installation

> The Wazuh Security Dashboards Plugin comes bundled by default as part of the Wazuh Dashboards distribution. Please refer to the [installation guide](https://documentation.wazuh.com/current/installation-guide/index.html).

Reviewer: ```suggestion
The Wazuh dashboard Security Plugin comes bundled by default as part of the Wazuh dashboard distribution. Please refer to the [installation guide](https://documentation.wazuh.com/current/installation-guide/index.html).
```

If you find a bug, or have a feature request, please don't hesitate to open an issue in this repository.

For more information, see [project website](https://wazuh.com/) and [documentation](https://documentation.wazuh.com/current/index.html). If you need help and are unsure where to open an [issue](https://github.com/wazuh/wazuh-security-dashboards-plugin/issues).

## Security

> If you discover a potential security issue in this project we ask that you notify AWS/Amazon Security via our [vulnerability reporting page](https://github.com/wazuh/wazuh-security-dashboards-plugin/issues/new/choose). Please do **not** create a public GitHub issue.

Reviewer 2: Shouldn't you mention the SECURITY.md file instead?

Author: This pr is to update the base branch as it was currently in 4.12.0.
